### PR TITLE
allow to remove MergeCell

### PR DESCRIPTION
**Pull request by zxdvd**
# PR Details

This PR exposes a method to remove a MergeCell.

## Description
I needs to flat all mergecells in excel. I've tried to get the area of a mergecell and use `SetCellValue` to set each cell in this area but it didn't work.

I think a `RemoveMergeCell` is needed.

## Related Issue
#527 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: cellmerged.go [hunk 1]
```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// RemoveMergeCell removes a MergeCell.
+func (f *File) RemoveMergeCell(sheet string, m MergeCell) error {
```
Comment (xuri) [inline cellmerged.go]: I propose to use `UnmergeCells` as the function name.

File: cellmerged.go [hunk 2]
```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
...
+			ref := xlsx.MergeCells.Cells[i].Ref
+			if ref == m[0] {
+				f.deleteMergeCell(xlsx, i)
+				break
+			}
+		}
+	}
+	return nil
+}
+
```
Comment (xuri) [inline cellmerged.go]: Please add the tests to cover your changes.

File: cellmerged.go (hunk 1 above)
Comment (zxdvd) [reply cellmerged.go]: I think you mean `UnmergeCell`? It unmerges a single mergecell.

File: cellmerged.go (hunk 2 above)
Comment (zxdvd) [reply cellmerged.go]: ok.

File: cellmerged.go [hunk 3]
```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// UnmergeCell unmerge a MergeCell.
+func (f *File) UnmergeCell(sheet string, m MergeCell) error {
```
Comment (xuri) [inline cellmerged.go]: After further thinking, I propose that the `UnmergeCell` and [`MergeCell`](https://godoc.org/github.com/360EntSecGroup-Skylar/excelize#File.MergeCell) functions use the same formal parameters to maintain semantics. The [`MergeCell`](https://godoc.org/github.com/360EntSecGroup-Skylar/excelize#MergeCell) structure just used to get merged cells.

Comment (zxdvd) [reply cellmerged.go]: Ok. I didn't find the MergeCell function in cell.go. Maybe the `MergeCell` will be moved into cellmerged.go in the future or move `UnmergeCell` to cell.go?

Comment (xuri) [reply cellmerged.go]: Yeah right, the `MergeCell` should be moved into `cellmerged.go`.

**Pull request merged**